electroncash/tests/test_consolidate.py

```diff
@@ -0,0 +1,247 @@
+import math
+import unittest
+from unittest.mock import Mock
+
+from .. import consolidate
+from ..address import Address
+
+TEST_ADDRESS: Address = Address.from_string(
+    "ecash:qr3l6uufcuwm9prgpa6cfxnez87fzstxesp7ugp0ez"
```

Comment:
```suggestion
    "bitcoincash:qr3l6uufcuwm9prgpa6cfxnez87fzstxescngr64l4"
```

Without this change, the test fails because Electron Cash cannot decode the address.

You can run this test with the following command (from the root of the project): 
`python -m electroncash.tests.test_consolidate`